src/nurse/models.py

```diff
@@ -74,3 +74,18 @@ def __str__(self):
     def is_valid(self):
         """Returns true if the prescription is still valid (e.g. hasn't expired)."""
         return self.start_date <= datetime.now().date() <= self.end_date
+
+
+class UserOneSignalProfileManager(models.Manager):
+    def subscription_ids_for_users(self, users):
+        subscription_ids = list(
+            self.filter(user__in=users).values_list("subscription_id", flat=True)
+        )
+
+        return [str(sub_id) for sub_id in subscription_ids]
```

Comment:
Can we postpone the evaluation? I assumed a Manager should return a QuerySet

Reply:
done